docker-entrypoint-initdb.d/install_timescaledb.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+TS_TELEMETRY='basic'
+if [ "${TELEMETRY}" == "off" ]; then
```

Comment:
I think we should prefix our TimescaleDB-specific env variables in the same way the postgres image does it. So, we'd have `TIMESCALEDB_TELEMETRY` to make clear that this concerns TimescaleDB.

I think we should also document this variable in the README.

Reply:
done